unit-tests/entry_test.py

```diff
@@ -0,0 +1,8 @@
+##############################################################################
+# (c) Crown copyright Met Office. All rights reserved.
+# For further details please refer to the file COPYRIGHT
+# which you should have received as part of this distribution
+##############################################################################
+# We do not unit test the application entry points as they are far too complex
+# for it. They would require too many fixtures and mocks to test. At that
+# point our unit test would have become a system test.
```

Comment:
Why do we have the test files if we aren't going to test anything?  I suppose it just signals to anyone reading the source that we have intentionally not included anything for the corresponding module?

Reply:
Exactly this. It short circuits people demanding to know why there are no tests for this module. It's like "This page intentionally left blank."